Missing file specifications

bash-4.3$ make
mkdir -p build/
nuget restore SteamAuth/SteamAuth/SteamAuth.sln
MSBuild auto-detection: using msbuild version '14.0' from '/usr/lib/mono/xbuild/14.0/bin'.
Restoring NuGet package Newtonsoft.Json.12.0.3.
Adding package 'Newtonsoft.Json.12.0.3' to folder '/home/vita/Desktop/Steam-authenticator/steamguard-cli/SteamAuth/SteamAuth/packages'
Added package 'Newtonsoft.Json.12.0.3' to folder '/home/vita/Desktop/Steam-authenticator/steamguard-cli/SteamAuth/SteamAuth/packages'

NuGet Config files used:
    /home/vita/.config/NuGet/NuGet.Config

Feeds used:
    /home/vita/.nuget/packages/
    https://api.nuget.org/v3/index.json

Installed:
    1 package(s) to packages.config projects
echo "NEWTONSOFT_JSON_PATH="
NEWTONSOFT_JSON_PATH=
mcs -target:library -out:build/SteamAuth.dll -r: SteamAuth/SteamAuth/APIEndpoints.cs SteamAuth/SteamAuth/AuthenticatorLinker.cs SteamAuth/SteamAuth/Confirmation.cs SteamAuth/SteamAuth/SessionData.cs SteamAuth/SteamAuth/SteamGuardAccount.cs SteamAuth/SteamAuth/SteamWeb.cs SteamAuth/SteamAuth/TimeAligner.cs SteamAuth/SteamAuth/UserLogin.cs SteamAuth/SteamAuth/Util.cs SteamAuth/SteamAuth/Properties/AssemblyInfo.cs
error CS2005: Missing file specification for `/r:' option
makefile:4: fallo en las instrucciones para el objetivo 'all'
make: *** [all] Error 1

bash-4.3$ git clone --recursive https://github.com/dyc3/steamguard-cli.git
Clonando en 'steamguard-cli'...
remote: Enumerating objects: 61, done.
remote: Counting objects: 100% (61/61), done.
remote: Compressing objects: 100% (33/33), done.
remote: Total 494 (delta 22), reused 57 (delta 21), pack-reused 433
Recibiendo objetos: 100% (494/494), 103.08 KiB | 1.26 MiB/s, listo.
Resolviendo deltas: 100% (283/283), listo.
Submódulo 'SteamAuth' (https://github.com/dyc3/SteamAuth.git) registrado para ruta 'SteamAuth'
Clonando en '/home/vita/steamguard-cli/SteamAuth'...
remote: Enumerating objects: 361, done.        
remote: Total 361 (delta 0), reused 0 (delta 0), pack-reused 361        
Recibiendo objetos: 100% (361/361), 85.47 KiB | 1.22 MiB/s, listo.
Resolviendo deltas: 100% (233/233), listo.
Ruta de submódulo 'SteamAuth': check out realizado a 'e0619528fbe8f4c6f74135283b3f991219e73cf4'
bash-4.3$ cd steamguard-cli/
bash-4.3$ make
mkdir -p build/
nuget restore SteamAuth/SteamAuth/SteamAuth.sln
MSBuild auto-detection: using msbuild version '14.0' from '/usr/lib/mono/xbuild/14.0/bin'.
Restoring NuGet package Newtonsoft.Json.12.0.3.
Adding package 'Newtonsoft.Json.12.0.3' to folder '/home/vita/steamguard-cli/SteamAuth/SteamAuth/packages'
Added package 'Newtonsoft.Json.12.0.3' to folder '/home/vita/steamguard-cli/SteamAuth/SteamAuth/packages'

NuGet Config files used:
    /home/vita/.config/NuGet/NuGet.Config

Feeds used:
    /home/vita/.nuget/packages/
    https://api.nuget.org/v3/index.json

Installed:
    1 package(s) to packages.config projects
echo "NEWTONSOFT_JSON_PATH="
NEWTONSOFT_JSON_PATH=
mcs -target:library -out:build/SteamAuth.dll -r: SteamAuth/SteamAuth/APIEndpoints.cs SteamAuth/SteamAuth/AuthenticatorLinker.cs SteamAuth/SteamAuth/Confirmation.cs SteamAuth/SteamAuth/SessionData.cs SteamAuth/SteamAuth/SteamGuardAccount.cs SteamAuth/SteamAuth/SteamWeb.cs SteamAuth/SteamAuth/TimeAligner.cs SteamAuth/SteamAuth/UserLogin.cs SteamAuth/SteamAuth/Util.cs SteamAuth/SteamAuth/Properties/AssemblyInfo.cs
error CS2005: Missing file specification for `/r:' option
makefile:4: fallo en las instrucciones para el objetivo 'all'
make: *** [all] Error 1

I tryed everyhing but same output. Then I tryed this in makefile:
$(eval NEWTONSOFT_JSON_PATH=/home/vita/steamguard-cli/SteamAuth/SteamAuth/packages/Newtonsoft.Json.12.0.3/lib/net45/Newtonsoft.Json.dll)

bash-4.3$ make
mkdir -p build/
nuget restore SteamAuth/SteamAuth/SteamAuth.sln
MSBuild auto-detection: using msbuild version '14.0' from '/usr/lib/mono/xbuild/14.0/bin'.
Ya se han instalado todos los paquetes mostrados en 'packages.config'.
echo "NEWTONSOFT_JSON_PATH=/home/vita/steamguard-cli/SteamAuth/SteamAuth/packages/Newtonsoft.Json.12.0.3/lib/net45/Newtonsoft.Json.dll"
NEWTONSOFT_JSON_PATH=/home/vita/steamguard-cli/SteamAuth/SteamAuth/packages/Newtonsoft.Json.12.0.3/lib/net45/Newtonsoft.Json.dll
mcs -target:library -out:build/SteamAuth.dll -r:/home/vita/steamguard-cli/SteamAuth/SteamAuth/packages/Newtonsoft.Json.12.0.3/lib/net45/Newtonsoft.Json.dll SteamAuth/SteamAuth/APIEndpoints.cs SteamAuth/SteamAuth/AuthenticatorLinker.cs SteamAuth/SteamAuth/Confirmation.cs SteamAuth/SteamAuth/SessionData.cs SteamAuth/SteamAuth/SteamGuardAccount.cs SteamAuth/SteamAuth/SteamWeb.cs SteamAuth/SteamAuth/TimeAligner.cs SteamAuth/SteamAuth/UserLogin.cs SteamAuth/SteamAuth/Util.cs SteamAuth/SteamAuth/Properties/AssemblyInfo.cs
error CS0009: Metadata file `/home/vita/steamguard-cli/SteamAuth/SteamAuth/packages/Newtonsoft.Json.12.0.3/lib/net45/Newtonsoft.Json.dll' does not contain valid metadata
Compilation failed: 1 error(s), 0 warnings
makefile:4: fallo en las instrucciones para el objetivo 'all'
make: *** [all] Error 1

--- a/makefile
+++ b/makefile
@@ -1,10 +1,9 @@
-﻿$(eval SHELL:=/bin/bash)
- 
-NEWTONSOFT_JSON_PATH="./SteamAuth/SteamAuth/packages/Newtonsoft.Json.12.0.3/lib/net45/Newtonsoft.Json.dll"
+$(eval SHELL:=/bin/bash)
 
 all: Program.cs
 	mkdir -p build/
 	nuget restore SteamAuth/SteamAuth/SteamAuth.sln
+	$(eval NEWTONSOFT_JSON_PATH=$(shell find -name Newtonsoft.Json.dll | grep \/net45\/ | sort -t. -k 1,1n -k 2,2n -k 3,3n -k 4,4n | tail -n 1))
 	mcs -target:library -out:build/SteamAuth.dll -r:$(NEWTONSOFT_JSON_PATH) SteamAuth/SteamAuth/APIEndpoints.cs SteamAuth/SteamAuth/AuthenticatorLinker.cs SteamAuth/SteamAuth/Confirmation.cs SteamAuth/SteamAuth/SessionData.cs SteamAuth/SteamAuth/SteamGuardAccount.cs SteamAuth/SteamAuth/SteamWeb.cs SteamAuth/SteamAuth/TimeAligner.cs SteamAuth/SteamAuth/UserLogin.cs SteamAuth/SteamAuth/Util.cs SteamAuth/SteamAuth/Properties/AssemblyInfo.cs
 	cp $(NEWTONSOFT_JSON_PATH) build/
 	mcs -out:build/steamguard -r:build/SteamAuth.dll -r:build/Newtonsoft.Json.dll -r:/usr/lib/mono/4.5/System.Security.dll Program.cs Manifest.cs AssemblyInfo.cs Utils.cs

--- a/makefile
+++ b/makefile
@@ -1,10 +1,10 @@
 ﻿$(eval SHELL:=/bin/bash)
+ 
+NEWTONSOFT_JSON_PATH="./SteamAuth/SteamAuth/packages/Newtonsoft.Json.12.0.3/lib/net45/Newtonsoft.Json.dll"
 
 all: Program.cs
 	mkdir -p build/
 	nuget restore SteamAuth/SteamAuth/SteamAuth.sln
-	$(eval NEWTONSOFT_JSON_PATH=$(shell find -name Newtonsoft.Json.dll | grep \/net45\/ | sort -t. -k 1,1n -k 2,2n -k 3,3n -k 4,4n | tail -n 1))
-	echo "NEWTONSOFT_JSON_PATH=$(NEWTONSOFT_JSON_PATH)"
 	mcs -target:library -out:build/SteamAuth.dll -r:$(NEWTONSOFT_JSON_PATH) SteamAuth/SteamAuth/APIEndpoints.cs SteamAuth/SteamAuth/AuthenticatorLinker.cs SteamAuth/SteamAuth/Confirmation.cs SteamAuth/SteamAuth/SessionData.cs SteamAuth/SteamAuth/SteamGuardAccount.cs SteamAuth/SteamAuth/SteamWeb.cs SteamAuth/SteamAuth/TimeAligner.cs SteamAuth/SteamAuth/UserLogin.cs SteamAuth/SteamAuth/Util.cs SteamAuth/SteamAuth/Properties/AssemblyInfo.cs
 	cp $(NEWTONSOFT_JSON_PATH) build/
 	mcs -out:build/steamguard -r:build/SteamAuth.dll -r:build/Newtonsoft.Json.dll -r:/usr/lib/mono/4.5/System.Security.dll Program.cs Manifest.cs AssemblyInfo.cs Utils.cs

